<!--
# Copyright (c) 2023, ANTOBOT LTD.
# All rights reserved.

# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
# Description:   Launcher for the heading node written in C++ 
# Contact: soyoung.kim@antobot.ai
# # # #  # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-->

<launch>
  <node pkg="antobot_heading" type="heading_node" name="heading_node" output="screen"> 
    <param name="gps_topic" type="string" value="/antobot_gps" />
    <param name="imu_topic" type="string" value="/imu/data" />
    <param name="odometry_topic" type="string" value="/odometry/filtered" />
    <param name="wheel_odometry_topic" type="string" value="/antobot_robot/odom" />
  </node>
</launch> 
